src/views.rs

```diff
@@ -0,0 +1,18 @@
+use crate::crd::model::Model;
+use crate::{Controller, Gordo};
+use actix_web::{http::StatusCode, web, HttpRequest, HttpResponse};
+
+// Simple health check endpoint
+pub async fn health(_req: HttpRequest) -> HttpResponse {
+    HttpResponse::new(StatusCode::OK)
+}
+
+// List current gordos
+pub async fn gordos(data: web::Data<Controller>, _req: HttpRequest) -> web::Json<Vec<Gordo>> {
+    web::Json(data.gordo_state().await)
```

Comment:
You want to make it even more awesome? Then set cache headers! E.g. `max-age=2`, and calculate the etag as a hash of the json! Do you think rust can do that? I dont know...

Reply:
Rust can do _anything_ :stuck_out_tongue_winking_eye:  But ya. `gordo_state()` essentially has the current state held immediately, because polling happens in the background continuously. Testing the response of this view in release mode averages a response time of 0.0002. Is it worth adding caching logic?

Reply:
I dont want the caching logic, just the cache **headers**  so it is easy to add a cache (e.g. at latigo's side) later. But if you prefer we can do it later.